src/components/variables.ts

```diff
@@ -12,8 +13,11 @@ export const palette = {
   whiteblue: '#E4F3FB',
   footerblue: '#01A5EC',
   orange: '#F0BA8C',
+  orange2: '#FFD79B',
```

Comment:
I would name the oranges with more meaningfull name than a number

Reply:
They are now the same names as in the figma, I thought that would be the most intuitive way of naming them